tests/suite/custom_resources_utils.py

```diff
@@ -697,11 +713,23 @@ def create_v_s_route_from_yaml(custom_objects: CustomObjectsApi, yaml_manifest,
     with open(yaml_manifest) as f:
         dep = yaml.safe_load(f)
 
+    return create_v_s_route(custom_objects, dep, namespace)
+
+def create_v_s_route(custom_objects: CustomObjectsApi, vsr, namespace) -> str:
```

Comment:
there's already a function called `create_v_s_route_from_yaml` in this file, which so no need to read a yaml in test and pass contents to a new function.